debian/rules

```diff
@@ -18,10 +18,10 @@ export DEB_BUILD_MAINT_OPTIONS=hardening=-stackprotector
 	dh $@ 
 
 override_dh_auto_build:
-	mvn package -DskipTests
+	mvn package -DskipTests -Dllvm.backend.prefix=/usr/lib/kframework -Dllvm.backend.destdir=$(shell pwd)/debian/kframework
 
 override_dh_auto_install:
-	DESTDIR=debian/kframework PREFIX=/usr src/main/scripts/package
+	DESTDIR=$(shell pwd)/debian/kframework PREFIX=/usr src/main/scripts/package
```

Comment:
Hmmm, this one doesn't make sense to me. Why would we want to use the full path to the directory where we're doing the packaging when building the package? That seems like it could potentially break other people's systems.

Reply:
it's necessary because the working directory when this variable gets used is actually something different. It is harmless, though, because DESTDIR does not get used after packaging finishes.